packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,26 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private getTitle() {
+        // Either get the title from the document, or return the default title
+        let title = "Blueprint - Documentation";
+        const docsTitle = document.getElementsByClassName("docs-title")[0].textContent;
+        // Check to make sure the textContent of docsTitle has a useable value
+        if (docsTitle !== undefined && docsTitle !== null) {
+            // Only change the title if we aren't on the index homepage
+            if (this.state.activePageId !== "blueprint" && this.state.activeSectionId !== "blueprint") {
+                title = `${docsTitle} - Blueprint`;
+            }
+        }
+        return title;
+    }
+
+    private updateTitle() {
+        // Update the page <title> when the hash changes
```

Comment:
This comment would make more sense where this function is _invoked_, if it's necessary at all. The function itself cannot guarantee that it will "update the page <title> [only] when the hash changes."